### PR TITLE
Hotfix falsy/null works

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: app/client/templates/application/add.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ Template.saved.events({
     var applicationValues = AutoForm.getFormValues('insertApplicationForm');
 
     // Update values
-    Applications.update(this._id, applicationValues.updateDoc, function(error, response) {
+    Meteor.call('saveApplication', this._id, applicationValues.updateDoc, function(error, response) {
```
Comment (varffs) [inline app/client/templates/application/add.js:27]: @mixcas what is this line change about? Why now the method call?
i suck u know

**Comment by mixcas** [comment]
@varffs well, i noticed that that call was wrong and it was relevant to fix the original issue so i fixed both things

File: app/client/templates/application/add.js (hunk 1 above)
Comment (varffs) [inline app/client/templates/application/add.js:27]: wtf not helpful to edit my comment and delete yours

**Comment by varffs** [comment]
Oh ok so comment comments vs line comment replies? Or maybe you cant reply to a line comment...

I think you should have split that change into another PR. Not sure that your change to method is correct. We should generally avoid calling to the server unless there are validations or functions we need to call to the server for. Meteor will push the data up the wire in its own sweet time if we make the save local. So why can't you do your removing of invalid null sections on the client?

**Comment by varffs** [comment]
@mixcas even tho this merged can you answer my question above

**Comment by mixcas** [comment]
@varffs 

Yeah, maybe splitting up could have been a good idea, I actually split it into different commits for this reason. I wasn't really sure how to approach it.

About the method change. I'm confused about this now. All I've read so far recommends methods for any kind of database modifications, more if it need validation. In this case we make sure the application belongs to the user.

Good point about cleaning it on the client. Is it better? same?

**Comment by varffs** [comment]
I've not seen anything that recommends using methods for db stuff over local. The obvious exception is anything that needs higher privilege than the user has or has to be on the server to function. I was under the impression that pub/sub it to be used as much as possible to minimize both network traffic and server load.

Probably better to clean on the client anyhow as again minimizing server load [even if extremely minimal why not let the user do the cpu cycles]

**Comment by mixcas** [comment]
Where should we move this discussion to? It's important. Wish u were here @varffs  <3

**Comment by varffs** [comment]
Trello on the meteor boil card or something?

**Pull request merged**